components/generic_modal.scss

```diff
@@ -0,0 +1,122 @@
+@charset 'UTF-8';
+
+.modal {
```

Comment:
Is this extra layer needed or can we start with `.GenericModal`? I think `.modal` is part of react-bootstrap, so whenever we get rid of that, this would need to go away anyway.